content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -54,6 +57,9 @@ Check if it worked:
 
 ```shell
 ls jobs/
+```
+
+```
```

Comment:
please  add `The output is similar to this:` as per the guidelines before any output